contracts/Strategy.sol

```diff
@@ -152,73 +179,55 @@ contract Strategy is BaseStrategy {
     }
 
     function adjustPosition(uint256 _debtOutstanding) internal override {
-        if (alreadyAdjusted) {
-            alreadyAdjusted = false; // reset for next time
+        if (
+            !forceCooldown &&
+            _checkCooldown() != CooldownStatus.None &&
+            balanceOfStkAave() > dustThreshold
```

Comment:
small gas savings. if you expect forceCooldown to be false most of the time, put it at the back so it doesn't need to read from it unnecesarily